include/dlaf/lapack_tile.tpp

```diff
@@ -43,6 +43,18 @@ dlaf::BaseType<T> lantr(lapack::Norm norm, blas::Uplo uplo, blas::Diag diag,
   return lapack::lantr(norm, uplo, diag, a.size().rows(), a.size().cols(), a.ptr(), a.ld());
 }
 
+template <class T, Device device>
+long long potrfInfo(blas::Uplo uplo, const Tile<T, device>& a) {
+  if (a.size().rows() != a.size().cols()) {
+    throw std::invalid_argument("Error: POTRF: A is not square.");
+  }
+
+  auto info = lapack::potrf(uplo, a.size().rows(), a.ptr(), a.ld());
+  assert(info >= 0);
+
+  return info;
+}
+
```

Comment:
We want to keep info/non-info variants together, but I would suggest to keep the alphabetic order. So, I would keep this after the non-info variant, but please let me know if you moved it for any specific reason.